apps/notifications/src/panel/templates/gridicons.jsx

```diff
@@ -1,354 +1,348 @@
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-notice':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Notice</title>
-						<g>
-							<path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm1 15h-2v-2h2v2zm0-4h-2l-.5-6h3l-.5 6z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-time':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Time</title>
-						<g>
-							<path d="M12 4c4.41 0 8 3.59 8 8s-3.59 8-8 8-8-3.59-8-8 3.59-8 8-8m0-2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm3.8 13.4L13 11.667V7h-2v5.333l3.2 4.266 1.6-1.2z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-reply':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Reply</title>
-						<g>
-							<path d="M9 16h7.2l-2.6 2.6L15 20l5-5-5-5-1.4 1.4 2.6 2.6H9c-2.2 0-4-1.8-4-4s1.8-4 4-4h2V4H9c-3.3 0-6 2.7-6 6s2.7 6 6 6z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-arrow-up':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Arrow Up</title>
-						<g>
-							<path d="M13 20V7.83l5.59 5.59L20 12l-8-8-8 8 1.41 1.41L11 7.83V20h2z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-arrow-down':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Arrow Down</title>
-						<g>
-							<path d="M11 4v12.17l-5.59-5.59L4 12l8 8 8-8-1.41-1.41L13 16.17V4h-2z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-arrow-left':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Arrow Left</title>
-						<g>
-							<path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-arrow-right':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Arrow Left</title>
-						<g>
-							<path d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8-8-8z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-cross':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Cross</title>
-						<g>
-							<path d="M18.36 19.78L12 13.41l-6.36 6.37-1.42-1.42L10.59 12 4.22 5.64l1.42-1.42L12 10.59l6.36-6.36 1.41 1.41L13.41 12l6.36 6.36z" />
-						</g>
-					</svg>
-				);
-
-			case 'gridicons-cog':
-				return (
-					<svg { ...sharedProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-						<title>Open notification settings</title>
-						<g>
+	switch ( gridIcon ) {
+		default:
+			return <svg height={ size } width={ size } />;
+
+		case 'gridicons-checkmark':
```

Comment:
I'm not sure why this gridicon doesn't have a title, but I'm also not seeing it used anywhere.